istqb-ct-genai-complete-guide.md

```
@@ -0,0 +1,1630 @@
+### 0.3 試験概要（v1.0）
+
+> | 項目 | 内容 |
+> | --- | --- |
+> | **問題数** | 40問 |
+> | **合格点** | 30点（46点満点）65% |
+> | **試験時間**<br>（英語非母語者） | 60分<br>(+25% = 75分) |
+> | **前提条件** | CTFL（必須） |
+> | **最高認知レベル** | K3（K4は出題なし） |
+> | **学習時間**（公認研修） | 最低13.6時間（5章） |
+> | **正式リリース** | 2025年7月25日 |
+> 注：CT-AI（47点満点・K4まで）より試験は易しめ
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)